Add support for <abbr> title & fix html detection

xal opened the pull request:
Changes for #6 & #7

xal committed:
Merge pull request #2 from data2viz/master

Pull changes from original repository

xal committed:
#7 add support for <abbr> title attribute

xal committed:
#6 - dont recognise java code like "client.request<String> {" as html

xal committed:
Merge commit 'ececf131eeaa7806b9a4277756e17bed64f4f5f0' into fix_issues

gzoritchak commented:
The issue #7 is not about the specific case of `abbr`'s title attribute. It is a generic one about how the plugin must convert HTML element attributes in Kotlinx.html. 
Please consider a more generic solution to this issue.

xal commented:
So, all attributes should be moved from constructor inside `{}` block? Ok, I will change

xal committed:
#7 Change attributes conversion. From constructor parameters to { } body.
Class attribute special conversion (from simple string to splitter set of strings) changed to support new attributes conversion

xal commented:
#7 Change attributes conversion. From constructor parameters to { } body. 
Class attribute special conversion (from simple string to splitter set of strings) changed to support new attributes conversion
